pages/bookmark-page.js

```diff
@@ -123,9 +159,21 @@ const StyledBarrelButton = styled.button`
   border: none;
   margin-top: 3px;
   background-color: rgba(31, 31, 35, 1);
-  :hover {
-    border-bottom: 3px solid white;
-    padding-top: 3px;
+`;
+
+const WineCellarButton = styled.button`
+  font-size: 16px;
+  display: flex;
+  align-items: center;
+  border-radius: 9px;
+  border: none;
+  padding-left: 4px;
+  background-color: rgba(31, 31, 34, 1);
+  color: white;
+  transition: 0.1s;
+  :active {
+    border-color: rgba(89, 199, 72, 1);
```

Comment:
For future reference: perhaps it would be beneficial to you to consider using variables defined in your GlobalStyles for colors/etc. It's much easier in terms of handling themes, etc.